src/containers/Experiment/ProcessingInformationCell.js

```diff
@@ -0,0 +1,394 @@
+
```

Comment:
This file seems to be moved from `src/containers/Experiment/SamplesTable.js`, right? If it is, I don't have to review it line by line.

Reply:
That's right, I didn't do any modifications there